nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -231,6 +231,12 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
```

Comment:
Apache NiFi uses Jackson for JSON processing throughout the framework.  In this particular case, Google Gson does not appear to provide any special features, so using the Jackson ObjectMapper would avoid introducing a different dependency for JSON handling.